llvm/test/CodeGen/MSP430/fake_use_float.ll

```diff
@@ -0,0 +1,31 @@
+; RUN: llc -stop-after=finalize-isel -filetype=asm -o - < %s | FileCheck %s
```

Comment:
```suggestion
; RUN: llc < %s | FileCheck %s
```
-filetype=asm is the default, so there's essentially no reason to ever use it. There's also no real reason to use a mir test here, and you can run to the end